addon-test-support/ember-cookies/clear-all-cookies.js

```diff
@@ -1,9 +1,19 @@
-export default function() {
+import { assert } from '@ember/debug';
+import { merge, assign as emberAssign } from '@ember/polyfills';
+import { isEmpty } from '@ember/utils';
+import { serializeCookie } from 'ember-cookies/utils/serialize-cookie';
+const assign = Object.assign || emberAssign || merge;
+
+export default function(options = {}) {
+  assert('Max-Age option cannot be set when clearing cookies', isEmpty(options.maxAge));
```

Comment:
I think this should also assert that `expires` cannot be specified?

Reply:
Also, it should assert that `httpOnly` is not specified. That would make no sense anyway of course but adding an assertion for it would help prevent people from running into situations that seem hard to understand and debug.

Reply:
Sure thing!